### PR TITLE
Django 1.9 support

### PR DESCRIPTION
**Pull request by eriktelepovsky**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *.pyc
 django_session_security.egg-info
 .ropeproject
+.idea
```
Comment (jpic) [inline .gitignore]: Could this be perhaps in a separate commit ?

**Comment by jpic** [comment]
Could you please update the travis yaml configuration to test against 1.8 and 1.9 and drop support for 1.4 ?

Thanks !

File: .gitignore (hunk 1 above)
Comment (eriktelepovsky) [inline .gitignore]: Is it so important?

Comment (jpic) [inline .gitignore]: Not so much xD

File: session_security/urls.py [hunk 2]
```diff
@@ -21,10 +21,10 @@
...
-urlpatterns = patterns('',
+urlpatterns = [
     url(
         'ping/$',
         PingView.as_view(),
         name='session_security_ping',
-    ),
-)
+    )
+]
```
Comment (jpic) [inline session_security/urls.py]: I think this is going to break with django < 1.8, which doesn't matter to me, but in that case those versions should be removed from travis.

Good job bro !!

**Comment by juhanar** [comment]
How's this coming along? I have a PR coming up (optional password lock in the warning dialog) for which I might like to have this merged first.

**Comment by jpic** [comment]
Apparently tests won't run on 1.8 and 1.9, can someone contribute support for the new test runner ?

I think basically you need to rename tests file like test_*.py

https://docs.djangoproject.com/en/1.9/releases/1.6/#discovery-of-tests-in-any-test-module

**Comment by jpic** [comment]
Closing in favor of #59, @eriktelepovsky it includes your commits, but i couldn't push new commits in your fork. Since you are part of the core dev team, could you maybe branch the @yourlabs repo instead of doing a fork in your @user ? This way, we'll be able to contribute commits on the same branch prior to merging ;)

**Comment by eriktelepovsky** [comment]
Yeah, sorry I didn't realize that.

**Comment by jpic** [comment]
NP, I have a lot of commits to push on that branch will do within an hour​
